Docker-Swarm-deployment/single-node/auditing-server/example-secrets/secrets/config.json

```diff
@@ -29,11 +29,9 @@
       "immudbRsTableName": "rsaudit",
       "immudbCatTableName": "auditingtable",
       "immudbAuthTableName": "table_auditing",
-      "immudbApdTableName" : "auditing_acl_apd",
```

Comment:
These apd auditing tables are there in config, please refer [config-dev.json](https://github.com/datakaveri/auditing-server/blob/5.0.0/example-configs/config-dev.json) (this is the source of truth + zookeeper config value) of each repos. the config-depl.json maybe outdated. @pranavv0